parameters/src/testnet3/genesis.rs

```diff
@@ -14,21 +14,21 @@
 // You should have received a copy of the GNU General Public License
 // along with the snarkVM library. If not, see <https://www.gnu.org/licenses/>.
 
-pub struct GenesisBytes;
+// pub struct GenesisBytes;
 
-impl GenesisBytes {
-    pub const fn load_bytes() -> &'static [u8] {
-        include_bytes!("./resources/block.genesis")
-    }
-}
+// impl GenesisBytes {
```

Comment:
Just testing the limits of how much I could remove before things break. This isn't necessary for the client.